support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.jsx

```diff
@@ -159,38 +177,69 @@ function PaperCheckoutForm(props: PropTypes) {
   const deliveryTitle = props.fulfilmentOption === HomeDelivery ? 'Where should we deliver your newspaper?' : `Where should we deliver your ${collectionOptionDescription}?`;
   const submissionErrorHeading = props.submissionError === 'personal_details_incorrect' ? 'Sorry there was a problem' :
     'Sorry we could not process your payment';
+  const title = `${getTitle(props.productOption)} ${fulfilmentOptionDescriptor.toLowerCase()}`;
+  const description = getShortDescription(props.productOption);
+  const productPrice = getProductPrice(
+    props.productPrices,
+    props.fulfilmentOption,
+    props.productOption,
+  );
+  const subsCardStartDates = props.useDigitalVoucher && props.fulfilmentOption === Collection ?
```

Comment:
```suggestion
 const isSubscriptionCard = props.useDigitalVoucher && props.fulfilmentOption === Collection;
  const subsCardStartDates =  isSubscriptionCard ?
```

and you can use that boolean in about 3 places below

Reply:
Nice! Thank you